### DISCUSSION TITLE
Add support for MDX

### DISCUSSION THREAD

**Discussion by zenorocha** [resend/react-email#573] [Ideas]
Inspiration:
* https://www.joshwcomeau.com/react/wonderful-emails-with-mjml-and-mdx/#composing-emails-with-mdx
* https://www.useplunk.com/markdown-to-email

Components needed:
* https://github.com/zenorocha/react-email/issues/53
* https://github.com/zenorocha/react-email/issues/52
* https://github.com/zenorocha/react-email/issues/49
* https://github.com/zenorocha/react-email/issues/48
* https://github.com/zenorocha/react-email/issues/42
* https://github.com/zenorocha/react-email/issues/13

**Comment by vcapretz** [comment]
Assigned myself for this one, going to work on it!

**Reply by ifielder** [reply]
Any update on this one?

**Reply by Dan86de** [reply]
Any update on support for mdx?

**Reply by bukinoshita** [reply]
Hey guys, we just merged this PR https://github.com/resendlabs/react-email/pull/707

**Reply by dngpng** [reply]
when will this be released?

**Reply by bukinoshita** [reply]
Just released https://www.npmjs.com/package/@react-email/markdown

**Reply by Dan86de** [reply]
Gr8 will check this one :)
